cfgov/unprocessed/apps/owning-a-home/js/explore-rates/tab.js

```diff
@@ -31,7 +31,9 @@ function _bindTabLink( tabGroup, tabContents ) {
    */
   function _tabLinkClicked( evt ) {
     const target = evt.target;
-    if ( target.tagName !== 'A' ) {
+
+    if ( target.tagName !== 'A' ||
+      target.classList.contains( 'tab-link' ) === false ) {
```

Comment:
Can this whole if statement simplified to `if ( !target.classList.contains( 'tab-link' ) ) {`?